scss/_patterns_search-and-filter.scss

```diff
@@ -114,11 +107,8 @@
       margin-bottom: 0;
       margin-left: -$sph-inner--small; // compensate for the left padding of the container
       margin-right: -$sp-x-large; // compensate for the space reserved for counter
```

Comment:
ideally, this should be expressed as the sum of the icon positioned there + the margins on either side.